custom_components/candy/config_flow.py

```diff
@@ -19,6 +19,7 @@
 
 STEP_DATA_SCHEMA = vol.Schema({
     vol.Required(CONF_IP_ADDRESS): str,
+    vol.Required(CONF_KEY_USE_ENCRYPTION): bool    
```

Comment:
I had problems with detecting encryption and i use it without encryption. I thin that i had problems vith oven and hub. 
Actually it shoud be also possible activate appliance with this candy API but activate work well just in case of Dishwasher.
I woud like to setup run remote oven but not working for me.

Reply:
I don't want to merge the PR with this change if we don't understand why it's needed and how other users of this integration are supposed to configure this option.

If this is too much work for you, I can understand that, in this case, I recommend keep using a forked version of this repo. But merging such changes into the main repo need to have better reasons and a bit more proof that it's really needed.

Reply:
Actually i can comment this for now. Because we need it just in case of setup mode.